django/mgmt/static/js/utils.js

```diff
@@ -175,30 +175,34 @@ function permission_formatter(response) {
     return response;
 }
 
-function resource_formatter(response, detail_base_url, delete_function, delete_base_url) {
+function resource_formatter(response, detail_base_url, delete_function, delete_base_url, is_admin) {
     // Base formatter for all resource bootstrap table plugins
     var output = [];
     for (var idx in response) {
         var actions_str = "<a type='button' class='btn btn-default btn-sm action-button' href=" + detail_base_url + response[idx] + ">";
         actions_str += "<span class='glyphicon glyphicon-pencil' aria-hidden='true'></span>  Details</a>";
-        actions_str += "<a type='button' class='btn btn-danger btn-sm action-button' href='javascript:void(0);' onclick='" + delete_function + "(\"" + delete_base_url + response[idx] + "\")'>";
-        actions_str += "<span class='glyphicon glyphicon-remove' aria-hidden='true'></span>  Delete</a>";
+        if (is_admin == 1){
+            actions_str += "<a type='button' class='btn btn-danger btn-sm action-button' href='javascript:void(0);' onclick='" + delete_function + "(\"" + delete_base_url + response[idx] + "\")'>";
+            actions_str += "<span class='glyphicon glyphicon-remove' aria-hidden='true'></span>  Delete</a>";
+        }
         output.push({"name": response[idx], "actions": actions_str})
     }
 
     return output
 }
 
-function channel_resource_formatter(response, detail_base_url, delete_function, delete_base_url, neuroglancer_url) {
+function channel_resource_formatter(response, detail_base_url, delete_function, delete_base_url, neuroglancer_url, is_admin) {
     // Base formatter for channel resource bottstrap table plugin
     var output = [];
     for (var idx in response) {
         var actions_str = "<a type='button' class='btn btn-default btn-sm action-button' href=" + detail_base_url + response[idx] + ">";
         actions_str += "<span class='glyphicon glyphicon-pencil' aria-hidden='true'></span>  Details</a>";
         actions_str += "<a type='button' class='btn btn-default btn-sm action-button' href=" + neuroglancer_url + response[idx] +  "','name':'" + response[idx] + "'}}}" + ">";
         actions_str += "<span class='glyphicon' aria-hidden='true'></span>  Neuroglancer</a>";
-        actions_str += "<a type='button' class='btn btn-danger btn-sm action-button' href='javascript:void(0);' onclick='" + delete_function + "(\"" + delete_base_url + response[idx] + "\")'>";
-        actions_str += "<span class='glyphicon glyphicon-remove' aria-hidden='true'></span>  Delete</a>";
+        if (is_admin == 1){
```

Comment:
We want to expect `is_admin` to be a boolean variable so we can use an if statement like so:

`if (is_admin === true) {`

We use triple equals to test for strict equality. Double equals tests for "truthiness" which means it's not `false`, `null`, or `undefined`.